cli_flags.go

```diff
@@ -299,6 +320,30 @@ func parseArgs() (*arguments, error) {
 				Destination: &args.site,
 				Sources:     cli.EnvVars("DD_HOST_PROFILING_SITE", "DD_SITE"),
 			},
+			&cli.StringFlag{
```

Comment:
Could we try using a custom flag for the SymbolEndpoints type?

This way we handle validation and parsing at once, and it wouldn't require us to parse the json twice.

Reply:
Validate that all fields from the json are set (!= "")